src/Pages/Repositories/AdminTaskTable/components/AdminTaskFilters.tsx

```diff
@@ -139,30 +139,60 @@ const AdminTaskFilters = ({ isLoading, setFilterData, filterData }: Props) => {
         );
       case 'Status':
         return (
-          <DropdownSelect_Deprecated
-            toggleAriaLabel='filter status'
-            toggleId='statusSelect'
-            ouiaId='filter_status'
+          <DropdownSelect
+            onSelect={(_, val) =>
+              setSelectedStatuses((prev) =>
+                selectedStatuses.includes(val as string)
+                  ? prev.filter((item) => item !== (val as string))
+                  : [...prev, val as string],
+              )
+            }
+            menuToggleProps={{
+              'aria-label': 'filter status',
+              id: 'statusSelect',
+            }}
+            multiSelect
+            dropDownItems={
+              statusValues.map((status) => ({
+                hasCheckbox: true,
+                value: status,
+                isSelected: selectedStatuses.includes(status),
+                children: status,
+                'data-ouia-component-id': `filter_${status}`,
+              })) as SelectOptionProps[]
+            }
             isDisabled={isLoading}
-            options={statusValues}
-            variant={SelectVariant.checkbox}
-            selectedProp={selectedStatuses}
-            setSelected={setSelectedStatuses}
-            placeholderText='Filter by status'
+            menuValue='Filter by Status'
```

Comment:
sentence case 🙈

Reply:
That was dangerous.. Maria would could have seen this... 😅